### ISSUE TITLE
CUSA01606 - Skullgirls 2nd Encore

### ISSUE THREAD

**Issue by StevenMiller123** [shadps4-emu/shadps4-game-compatibility#58]
### Game Name

Skullgirls 2nd Encore

### Game code

CUSA01606

### Game version

v1.00

### Used emulator's version (only released versions are acceptable)

v0.1.0

### Current status

Menus

### Error

[Render.Recompiler] <Error> translate.cpp:Translate:829: Unknown opcode V_READFIRSTLANE_B32 (465)
[Debug] <Critical> structured_control_flow.cpp:operator():826: Assertion Failed!
Shader translation has failed

### Description

The game successfully renders menus, though character sprites appear corrupted. When trying to enter any form of battle, the game will crash with the above errors. Adding libSceNgs2 to user/sys_modules is required for audio to work, and the provided log has it added. 

### Screenshots

<img width="641" alt="Skullgirls 2nd Encore Screenshot" src="https://github.com/shadps4-emu/shadps4-game-compatibility/assets/56742918/280ff61d-6b63-4268-b288-d4aa70b39c62">

### Log File

[Skullgirls 2nd Encore Log.txt](https://github.com/user-attachments/files/16075630/Skullgirls.2nd.Encore.Log.txt)

**Comment by StevenMiller123** [comment]
On emulator v0.2.0, the game no longer crashes when trying to enter battles. All ingame sprites render broken, as they did in v0.1.0.

![Screenshot 2024-08-15 185400](https://github.com/user-attachments/assets/ee038406-59a9-4e00-abb1-37d3f69c65fe)

The game's status should be changed to Ingame.
Log: [Skullgirls 2nd Encore log.txt](https://github.com/user-attachments/files/16631981/Skullgirls.2nd.Encore.log.txt)

**Comment by StevenMiller123** [comment]
On emulator v0.3.0, the game behaves identically to the previous emulator version.

No change in game status.
[shad_log.txt](https://github.com/user-attachments/files/17123343/shad_log.txt)

**Comment by StevenMiller123** [comment]
On v0.4.0, the rendering issues are fixed.
![image](https://github.com/user-attachments/assets/05688c7b-3dc5-4b7a-b5e8-84938529d040)
[shad_log.txt](https://github.com/user-attachments/files/17592289/shad_log.txt)

**Comment by DLADAnaconda** [comment]
Booted the game today with updates and dlc. Plays great but there is no sound/ ChadPS4 v0.5.0
![shadPS4_up5RXWLbVw](https://github.com/user-attachments/assets/f2e34418-374d-4b2c-8d59-1dfc71421f44), 
![shadPS4_6y00U0wf8t](https://github.com/user-attachments/assets/dbb29733-eacd-4865-ae1b-0473eea96bc3)
on
![shadPS4_a6K92nsPls](https://github.com/user-attachments/assets/38a7cbbd-3103-4c3c-b884-62097e421875)
[shad_log.txt](https://github.com/user-attachments/files/18255632/shad_log.txt)
![shadPS4_a6K92nsPls](https://github.com/user-attachments/assets/cd618b71-8aa4-44fd-b612-30c940420193)

**Comment by DLADAnaconda** [comment]
Tested the game with sys modules for firmware 11, is ps4 pro  @off
 and the game seems to play just fine, with audio.
[shad_log.txt](https://github.com/user-attachments/files/18271160/shad_log.txt)

**Comment by StevenMiller123** [comment]
@Hermiten audio issues tag can be removed, those problems were caused by missing sys_modules (see message from DLADAnaconda above)

**Comment by DLADAnaconda** [comment]
> @Hermiten audio issues tag can be removed, those problems were caused by missing sys_modules (see message from DLADAnaconda above)

Yes but they should add like a sys modules tag to the games also, or something to distinguish how a game should run/what makes the game playable.

**Comment by Hermiten** [comment]
That's true with discuss about this
